plugins/node.d/amavis

```diff
@@ -77,69 +77,75 @@ GPLv2
 
 =cut
 
-AMAVIS_LOG=${amavislog:-/var/log/mail/mail.info}
-LOGTAIL=${logtail:-logtail}
+mktempfile () {
+    mktemp -p /tmp/ $1
+}
+
+AMAVIS_LOG=${amavislog:-/var/log/maillog}
+LOGTAIL=${logtail:-/usr/local/bin/logtail}
 STATEFILE=$MUNIN_PLUGSTATE/amavis.offset
-mktemp_command="${mktemp_command:-/bin/mktemp}"
+AMAVISD=${amavisd:-/usr/local/sbin/amavisd}
 
 if [ "$1" = "autoconf" ]; then
-        if [ -f "${AMAVIS_LOG}" -a -n "${LOGTAIL}" -a -x "${LOGTAIL}" ] ; then
-		echo yes
-		exit 0
-	else
-		echo no
-		exit 0
-	fi
+    if [ ! -f "${AMAVIS_LOG}" ]; then
+        echo "no (\$AMAVIS_LOG was not found)"
```

Comment:
Are you sure, that you want to escape the `$` character? Probably you want to display the real filename instead of confusing the user with the internal variable names used by this plugin.